dependencies.yaml

```diff
@@ -940,8 +941,10 @@ dependencies:
           - &fastavro fastavro>=0.22.9
           - mmh3
           - nanoarrow
+          - pytest-env
           - hypothesis>=6.131.7
           - *numpy
+          - "nvtx>=0.2.1,<0.2.12"
```

Comment:
I thought `pyenv` was only needed for the narwhals test suite, not pylibcudf tests. We should draw that line clearly.

Also, `nvtx>=0.2.1,<0.2.12` is a run dependency of pylibcudf -- so we should not need to include it here as a test dependency, since it's already pinned in the run dependencies.

Reply:
Done in https://github.com/rapidsai/cudf/pull/18971/commits/9a29c5892aeadf6998b3862f02f06640bbfb83a7